pyroute2/ipdb/interface.py

```diff
@@ -779,9 +779,18 @@ def error(x):
             #
             # One simple way to work that around is to remove
             # secondaries first.
-            rip = sorted(removed['ipaddr'],
-                         key=lambda x: self['ipaddr'][x]['flags'],
```

Comment:
Ough. I just missed one stupid thing.

Look, there is `flags` byte field as well as uint32 `IFA_FLAGS`. Even if the NLA is not present (→ None), the normal field will be an unsigned int in the range 0…255. So where from come addresses with None-flags?

That should be a bug in IPDB then, I believe.